receiver/k8sclusterreceiver/internal/deployment/documentation.md

```diff
@@ -0,0 +1,38 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# k8s/deployment
+
+## Default Metrics
+
+The following metrics are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: false
+```
+
+### k8s.deployment.available
+
+Total number of available pods (ready for at least minReadySeconds) targeted by this deployment
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
+
+### k8s.deployment.desired
+
+Number of desired pods in this deployment
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
+
+## Resource Attributes
+
+| Name | Description | Values | Enabled |
+| ---- | ----------- | ------ | ------- |
+| k8s.deployment.name | The name of the Deployment. | Any Str | true |
+| k8s.deployment.uid | The UID of the Deployment. | Any Str | true |
+| k8s.namespace.name | The name of the namespace that the pod is running in. | Any Str | true |
+| opencensus.resourcetype | The OpenCensus resource type. | Any Str | true |
```

Comment:
Let's remove this body :) It's an OC conversion artifact and not supposed to be emitted in the first place

Reply:
This will break e2e, this is a breaking change. I think we should keep it for now, and turn it off across the whole receiver as disabled and eventually remove it as separate breaking changes.